mollie/api/resources/profiles.py

```diff
@@ -10,7 +10,9 @@ def get_resource_object(self, result):
         return Profile(result, self.client)
 
     def get(self, profile_id, **params):
-        if not profile_id or not profile_id.startswith(self.RESOURCE_ID_PREFIX):
+        if not profile_id or \
+                (not profile_id.startswith(self.RESOURCE_ID_PREFIX)
+                 and not profile_id == 'me'):
             raise IdentifierError(
                 "Invalid profile ID: '{id}'. A profile ID should start with '{prefix}'.".format(
```

Comment:
The error message for organizations also states that the ID could be "me". This one works the same but the error message misses that part.

Reply:
Done